src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CompilationTracker.cs

```diff
@@ -742,13 +735,8 @@ public ICompilationTracker WithCreateCreationPolicy(bool forceRegeneration)
                     skeletonReferenceCacheToClone: _skeletonReferenceCache);
             }
 
-            public ICompilationTracker WithDoNotCreateCreationPolicy(
-                bool forceRegeneration, CancellationToken cancellationToken)
+            public ICompilationTracker WithDoNotCreateCreationPolicy(CancellationToken cancellationToken)
```

Comment:
needs the cancellation token since we freeze the tracker, which means grabbing the syntax trees available at thsi point in time.